src/screens/ProfileScreen/elements/LinkAndSocialMedia.tsx

```diff
@@ -103,8 +106,9 @@ const LinkAndSocialMedia: React.FC<LinkProps> = ({username}) => {
 
 const styles = StyleSheet.create({
   linkAndSocialMediaTitle: {
-    fontWeight: '500',
-    fontSize: 14
+    fontFamily: fonts.inter[500],
+    fontSize: 14,
+    color: COLORS.white2
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The fontWeight and color properties have been replaced with fontFamily and a new color. This change provides more flexibility in terms of font weight and ensures consistency in color usage.
```diff
-    fontWeight: '500',
+    fontFamily: fonts.inter[500],
+    color: COLORS.white2
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->